deploy/crds/enterprise.splunk.com_licensemasters_crd.yaml

```diff
@@ -2646,6 +2646,29 @@ spec:
                                 type: object
                               appName:
                                 type: string
+                              auxPhaseInfo:
```

Comment:
@sgontla Do we need these changes for MC? OR should this be part of PR #571

Reply:
Yes, it comes by default for all the CRs, as it is one data structure that is used for App Deploy info across the places.